### PR TITLE
Ce 10474 prepare release

### PR DESCRIPTION
**Pull request by matthias-specht**
pls have a look at the 2 last commits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,20 +29,24 @@ This document explains how to install the CityEngine SDK and how to work with th
...
 * [Related Software Products](#related-software-products)
+* [General Software Requirements](#general-software-requirements)
 * [Release History and Changelog](#release-history-and-changelog)
 * [CityEngine Resources](#cityengine-resources)
 * [Issues and Contributions](#issues-and-contributions)
 * [Licensing](#licensing)
 
+## Downloads
+
+Download the SDK archive, the open-source examples in this repository and the example data from the Assets section on the [Releases](https://github.com/Esri/cityengine-sdk/releases/latest) page.
```
Comment (SGoetschi) [inline README.md:48]: English has two commas here: ... SDK archive, the ... repository, and ...

Comment (SGoetschi) [inline README.md:48]: I'd rather go with "You can download ..." than "Download ..."
After all the user does not need to download the SDK zips himself anymore.

**Review by SGoetschi — APPROVED** [review]
I pointed out three things I would change but it's fine like this as well.

**Comment by matthias-specht** [comment]
applied the changes and merged to develop.

**Pull request closed without merging**